membership/src/main/java/net/corda/v5/membership/NotaryInfo.java

```diff
@@ -48,4 +50,9 @@ public interface NotaryInfo {
      * @return The public key of the notary service, which will be a composite key of all notary virtual nodes keys.
      */
     @NotNull PublicKey getPublicKey();
+
+    /**
+     * @return boolean of whether it requires backchain verification.
+     */
+    @NotNull Boolean getBackchainRequired();
```

Comment:
Should this be `isBackchainRequired` to stick with normal java naming conventions for booleans?